### DISCUSSION TITLE
SSL certificate on if app installed on docker

### DISCUSSION THREAD

**Discussion by tathagata1428** [JulianPrieber/llc-docker#72] [Issues]
, has anyone been able to successfully set up Linkstack on Docker with an SSL certificate?"

**Comment by JulianPrieber** [comment]
You could use Cloudflare for that

**Comment by bcsanad** [comment]
Name your SSL certificate as server.pem and your SSL key as server.key and attach (bind) the folder where these two files are as a volume to /etc/ssl/apache2.

E.g.:    -v /etc/mycertificates:/etc/ssl/apache2

**Comment by Chase295** [comment]
Is there any more information´s. Its my first time with docker and so on.
